Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/datadonation/ui/DataDonationTestFragmentViewModel.kt

```diff
@@ -85,6 +109,42 @@ class DataDonationTestFragmentViewModel @AssistedInject constructor(
         }
     }
 
+    fun selectSafetyNetExceptionType(type: SafetyNetException.Type) {
+        currentSafetyNetExceptionTypeInternal.value = type
+    }
+
+    fun showSafetyNetErrorDialog() {
+        when (currentSafetyNetExceptionTypeInternal.value) {
+            SafetyNetException.Type.APK_PACKAGE_NAME_MISMATCH,
```

Comment:
We also have these same mappings in `SurveyConsentViewModel`. I think we should re-use them. Because if they change in the `ViewModel` and we forget to change them here, the TestMenu shows a different error message than in the one in the real usage.

Reply:
How?

Reply:
adjusted